bigbone/src/main/kotlin/social/bigbone/api/entity/data/Focus.kt

```diff
@@ -0,0 +1,17 @@
+package social.bigbone.api.entity.data
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Contains the coordinates to be used for smart thumbnail cropping.
+ * For more details, please have a look at https://docs.joinmastodon.org/api/guidelines/#focal-points
+ */
+data class Focus(
+    @SerializedName("x")
+    val x: Float,
+
+    @SerializedName("y")
+    val y: Float
+)
+
+fun Focus.asString(): String = "$x, $y"
```

Comment:
More of a style question than anything else: this would also be possible as a function in a companion object - do we want to favor one or the other? I'm not sure what effect this might have on the resulting `static` Java method (probably none?), but I just stumbled upon the import of `social.bigbone.api.entity.data.asString` without any mention of `Focus`.

Reply:
Let's prefer companion objects. But in this case it was not doable, as the companion object is static and I need to access instance members. Therefore, I have added a conversion function directly into the data class.